diff --git a/dist/index.js b/dist/index.js
index dc4f8326387c2112da6460500a3bf87a855ac80d..a91209f427b4e34488b0684488e72d16ded66462 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -2177,7 +2177,7 @@ function TOC({ headings, filePath }) {
   const config = useConfig();
   const tocRef = useRef7(null);
   const items = useMemo3(
-    () => headings.filter((heading) => heading.depth > 1),
+    () => headings.filter((heading) => heading.depth == 2),
     [headings]
   );
   const hasHeadings = items.length > 0;